cpp/test/Ice/custom/TestAMDI.cpp

```diff
@@ -4,18 +4,44 @@
 
 #include "TestAMDI.h"
 #include "Ice/Communicator.h"
+#include "TestHelper.h"
 
 using namespace std;
 using namespace Ice;
 using namespace Test;
 
+void
+TestIntfI::opShortArrayAsync(
+    pair<const int16_t*, const int16_t*> inSeq,
```

Comment:
don't we need to add padding here too?

Reply:
No, the padding is a function of the number of previous bytes in the request message (14 for the header, N for the operation name etc.).